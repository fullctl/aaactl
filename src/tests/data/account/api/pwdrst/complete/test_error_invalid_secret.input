{
  "client": "api_client_anon",
  "data": {
    "secret": "invalid",
    "password_new": "newpassword",
    "password_confirmation": "newpassword"
  }
}
