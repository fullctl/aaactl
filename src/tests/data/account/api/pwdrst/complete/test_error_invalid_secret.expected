{
  "status": 400,
  "response":{"data": [], "errors": {"secret": ["Invalid secret"]}, "profiling": {"queries": 0}}
}
